src/opnsense/scripts/OPNsense/CheckIP/service_test.php

```diff
@@ -0,0 +1,43 @@
+#!/usr/local/bin/php
+<?php
+/**
+ *    Copyright (C) 2018 Deciso B.V.
+ *
+ *    All rights reserved.
+ *
+ *    Redistribution and use in source and binary forms, with or without
+ *    modification, are permitted provided that the following conditions are met:
+ *
+ *    1. Redistributions of source code must retain the above copyright notice,
+ *       this list of conditions and the following disclaimer.
+ *
+ *    2. Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *
+ *    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ *    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ *    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ *    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *    POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+require_once("config.inc");
+require_once("interfaces.inc");
+require_once("util.inc");
+require_once("services.inc");
+
+  $service = (isset($argv[1])) ? trim($argv[1], " \n")    : '';
+$interface = (isset($argv[2])) ? trim($argv[2], " \n")    : 'wan';
+      $ipv = (isset($argv[3])) ? trim($argv[3], "ipv \n") : 4;
+
+echo      "Service: " . ($service ? $service : 'selected by interface and IP version') . 
+      ", Interface: " . $interface . 
+     ", IP Version: " . $ipv . 
+     ", IP Address: " . get_dyndns_ip($interface, $ipv, $service);
+?>
```

Comment:
when returning structured data, we usually use JSON.

Reply:
Changed to JSON